src/csharp/Grpc.nuspec

```diff
@@ -5,7 +5,7 @@
     <title>gRPC C#</title>
     <summary>C# implementation of gRPC - an RPC library and framework</summary>
     <description>C# implementation of gRPC - an RPC library and framework. See project site for more info.</description>
-    <version>0.5.0</version>
+    <version>0.5.0.1</version>
```

Comment:
That basically says it's still 0.5.0 release, but I fixed the package itself (and I have to bump the version somehow because the packages are read only once published).